### PR TITLE
Update the dashboard's assetsvc URIs to be namespaced.

### PR DESCRIPTION
**Pull request by absoludity**
Follows #1567 , Ref #1521 

Updates the frontend URIs to match the new assetsvc's namespaced URIs. Currently it just uses the config kubeapps namespace so that this can land. I'll continue by making this dependent on the selected namespace, when the feature flag is set.

### CODE REVIEW

File: dashboard/src/shared/Chart.ts [hunk 1]
```diff
@@ -4,49 +4,40 @@ import { IChart, IChartVersion } from "./types";
...
-  public static async exists(id: string, version: string, repo: string) {
-    try {
-      await axiosWithAuth.get(URL.api.charts.getVersion(`${repo}/${id}`, version));
-    } catch (e) {
-      return false;
-    }
-    return true;
-  }
-
   public static async listWithFilters(name: string, version: string, appVersion: string) {
```
Comment (andresmgot) [inline dashboard/src/shared/Chart.ts]: this function also requires to get the namespace, no?

File: dashboard/src/shared/Chart.ts [hunk 2]
```diff
@@ -4,49 +4,40 @@ import { IChart, IChartVersion } from "./types";
...
   }
 
-  public static async getSchema(id: string, version: string) {
-    const { data } = await axiosWithAuth.get<JSONSchema4>(URL.api.charts.getSchema(id, version));
+  public static async getSchema(namespace: string, id: string, version: string) {
+    const { data } = await axiosWithAuth.get<JSONSchema4>(URL.api.charts.getSchema(namespace, id, version));
     return data;
   }
 
-  public static async exists(id: string, version: string, repo: string) {
```
Comment (andresmgot) [inline dashboard/src/shared/Chart.ts]: why did you delete this function?
[edit] I see, apparently it's not being used

File: dashboard/src/shared/Chart.ts (hunk 1 above)
Comment (absoludity) [reply dashboard/src/shared/Chart.ts]: Indeed, updated, thanks.
